docs/site/home.js

```diff
@@ -20,6 +22,20 @@ angular
       templateUrl: 'site/home.html',
       controller: 'HomeCtrl'
     });
+
+    markdownConverterProvider.config({
+      extensions: [function () {
+        return [{
+          type: 'lang',
+          regex: '\\[([^\\]]+)\\]\\((\\/[^)]+)\\)',
```

Comment:
I guess we could trim them off safely? It's gross, but not as gross as that string, I'm pretty sure!